build/bazelutil/bazel-generate.sh

```diff
@@ -2,6 +2,6 @@
 
 set -exuo pipefail
 
-bazel run //:gazelle -- update-repos -from_file=go.mod -build_file_proto_mode=disable_global -to_macro=DEPS.bzl%go_deps -prune=true
-bazel run //pkg/cmd/generate-test-suites --run_under="cd $PWD && " > pkg/BUILD.bazel
-bazel run //:gazelle
+bazel run --symlink_prefix=.bazel/ //:gazelle -- update-repos -from_file=go.mod -build_file_proto_mode=disable_global -to_macro=DEPS.bzl%go_deps -prune=true
+bazel run --symlink_prefix=.bazel/ //pkg/cmd/generate-test-suites --symlink_prefix=.bazel/ --run_under="cd $PWD && " > pkg/BUILD.bazel
```

Comment:
If we're going to do this -- and I'm not sure that we should -- then this should go in `.bazelrc` so that it's common to all Bazel invocations. Otherwise you're going to be chasing around `bazel` invocations all the time. An example is `build/bazelutil/bazeltest.sh`, which we've already missed, and of course only more `bazel` invocations are going to be added over time.